assembl/static2/js/app/components/administration/landingPage/customizeHeader.jsx

```diff
@@ -1,137 +1,174 @@
 // @flow
```

Comment:
You should rename `customizeHeader` to `index` and put it in the `header` folder

Reply:
Moved it to be more consistent with the other components.